routes/index.js

```diff
@@ -1,4 +1,6 @@
+const users = require('./users');
```

Comment:
I think its better if this variable will be called:
`const users_routes = require('./users');`
I find it a bit clearer.